x11-misc/albert/albert-0.22.17.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10,11,12} )
+
+inherit cmake python-single-r1 xdg
+
+DESCRIPTION="A fast and flexible keyboard launcher"
+HOMEPAGE="https://albertlauncher.github.io/"
+
+PLUGINS_HASH="02755dfa0ef3f17c28a813355eb7db60642ee797"
+PYTHON_EXTENSIONS_COMMIT="2067bbb3d8fa5cfa5df2be9cada29a7e6715f07a"
+
+SRC_URI="
+	https://github.com/albertlauncher/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/albertlauncher/plugins/archive/${PLUGINS_HASH}.tar.gz -> ${PN}-plugins-${PLUGINS_HASH}.tar.gz
+	python-extensions? ( https://github.com/albertlauncher/python/archive/${PYTHON_EXTENSIONS_COMMIT}.tar.gz -> ${PN}-python-extensions-${PYTHON_EXTENSIONS_COMMIT}.tar.gz )
+"
+
+LICENSE="Albert-1.0"
```

Comment:
This is a falsehood and I object to it -- please revert.

At *minimum*, the software is not licensed under Albert-1.0, it is licensed under "bits and pieces of external GPL contributions, plus Albert-1.0".

Most likely, it is legally forbidden for a user to manually `git clone https://github.com/albertlauncher/albert`, then compile and install it in their $HOME -- because to do so would be to be aiding and abetting an ongoing copyright infringement. In that case, Gentoo should not be including an ebuild for it either.

This is NOT the same as an ordinary package that happens to be proprietary software and which is entirely fine for users to download and install -- and for which Gentoo has a legitimate use case to help users do that, even if it has to be mirror-restricted since Gentoo doesn't have the legal authority to redistribute copies and the users must go directly to the original source.

IMO the original "all-rights-reserved" was completely 100% correct, since this is the least we can do in order to indicate that the licensing situation is *significantly* more complicated than the upstream author "license" claims, and "all-rights-reserved" put the onus on users to investigate whether and how they can use that.

The current state of affairs is that even after @ulm added `RESTRICT="mirror bindist"` back, users are erroneously made to think that as long as they are okay with the contents of licenses/Albert-1.0, it's okay to install and use this software.